.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
We have a `.gitignore` in the app directory which should avoid node_modules being tracked. Do we need this one on top of it?

Reply:
I added it because for some reason my language server keeps creating it at the top level (mentioned it in the commit message).

I'll remove it if bothers you and look into configuring my language server to not write the debug.log in there.